### PR TITLE
Chapter 5 - Moderation

### PR DESCRIPTION
**Pull request by SaarthakDeshmukh**
Chapter 5 completed successfully.  :tada: ✅ :

### CODE REVIEW

File: 7-scrum-master-role/5-moderation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Chapter 5 - Moderation
+
+Moderation is unconventionally regarded as a means of conducting meetings or group activities in a non-controlling manner. It aims at making use of group knowledge by involving group members in the decision-making process and by focusing them on the purpose of the meeting. The moderation requires a specific basic attitude of the moderator and certain techniques. Let’s start with what and how to recognize a good event moderator.
```
Comment (udeysingh95) [inline 7-scrum-master-role/5-moderation.md:3]: Very scientific way of explaining moderation - how would you describe it to a 6 year old kid? Use that language.

File: 7-scrum-master-role/5-moderation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+4. Flexibility: No matter how well prepared the discussion guide is, or how familiar the moderator is with the topic, revelations coming from respondents create the need for additional or different question areas. Frequently, it is these bolts of brilliance that turn an ordinary group into an extraordinary group. However, it is up to the moderator to handle what pops up and work with it.
+5. Strong verbal skills: Effective verbal communication skills include more than just talking. Verbal communication encompasses both how you deliver messages and how you receive them. Moderators who can convey information clearly and effectively are highly valued by the team.
+6. Creating comfort and trust: When respondents feel comfortable, signs that a group is working are their body language, interest, involvement, and the number of times they laugh.
+7. Be Enthusiastic and Attentive: Have a high energy level, pay attention to participants and recognize group dynamics.
+
+## Scrum Master as Moderator
```
Comment (udeysingh95) [inline 7-scrum-master-role/5-moderation.md:39]: This is the heart of this chapter - and it comes almost at the end. 
Try to start with this and keep the knowledge revolving around SM as moderator. 
If you prefer to moderator as the core of this chapter and then quickly connect with SM role - as it is now - then that is fine as well - just give a justification for that - why did you choose to go that way? 
i would be interested to hear your thoughts then.

File: 7-scrum-master-role/5-moderation.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+**4. Make the speakers the center of attention:** As a moderator, you are not the center of attention. The speakers are. The best moderators know how to take a step back. To shut up when needed. Always be aware of the fact that you should keep what you say as short as possible. Summarise what the speakers said. Ask short questions and repeat the questions from the audience. That’s it.\
+**5. Pay attention:** Being an event moderator is hard work! You are the only one who knows for sure that you need to pay attention. You want to know what happens and you want to make sure you ask the right questions. For that, you need to pay attention.\
+**6. Be a host:** Finally, As a moderator, you are more than the person watching time. You are the person in charge of the session. Be attentive. Know who’s in front of you and make sure they have a productive session and that they get something out of the meeting.\
+Now let us look at some characteristics of a great moderator.
+
+## Characteristics of a great moderator
+
+The 7 most important qualities of a great moderator.
```
Comment (udeysingh95) [inline 7-scrum-master-role/5-moderation.md:29]: Points start right after the title. There is only a cover-up standing here - which is clear. Try to use para format to open up a section.

File: 7-scrum-master-role/5-moderation.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+1. Natural curiosity: Phrases like “I don’t know,” “I guess so” and “I’m not sure” create great opportunities to any true moderator. No accomplished moderator will allow any of these responses to be the final word – they need to know more, they need to understand the whys, and they have the skill to help respondents comfortably articulate their thoughts.
```
Comment (udeysingh95) [inline 7-scrum-master-role/5-moderation.md:31]: Flat information - how can we make it more engaging? I like the points given below - but Its hard to go in each and read them.

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
Good size of the file.
As I mentioned earlier as well, a coach or a SM should have good communication and teaching skills because its one of their regular instance.  They should be skilled/proficient in giving their message in the easiest way possible. I see this lacking a bit in these PRs. 
Theres nothing to worry about it - keep practising and you will improve it. We are all here to learn only. 
Its your sandbox to try and fail and learn - remember- fail fast and learn. ;)

File: 7-scrum-master-role/5-moderation.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+Moderation focuses on keeping the information and communication flow clear and accessible to all who participate. It aims at making use of group knowledge by involving group members in the decision-making process and by focusing them on the purpose of the meeting. It requires a specific basic attitude of the moderator and certain techniques. Let’s start with what and how to recognize a good event moderator.
+
+## Moderator
+
+A Moderator is someone who tries to help other people come to an agreement. They help the meeting group to achieve its goals in a focused manner. To guide the discussion in the proper direction, focus groups are typically led by at least one moderator. An event moderator is the master of ceremony of the event. He or she is there to make sure the speakers can do the best job and the audience gets the most out of the day or session. A moderator introduces speakers. He or she also makes sure the speakers stick to the time and the moderator asks and moderate questions. He or she is the connection between the different talks. Having a successful moderator can make a real difference in the effectiveness of a focus group. Let's see Scrum Master, as Moderator.
```
Comment (udeysingh95) [inline 7-scrum-master-role/5-moderation.md:7]: Very interesting question - how is a moderator different from a facilitator?

**Review by udeysingh95 — APPROVED** [review]

**Review by Vaibhavpratapsingh22 — APPROVED** [review]

File: 7-scrum-master-role/5-moderation.md (hunk 5 above)
Comment (SaarthakDeshmukh) [reply 7-scrum-master-role/5-moderation.md:7]: We can say a Facilitator is a person responsible for promoting a Process and a Moderator is more of a guide, helps the team to achieve its goals in a focused manner. For ex, SM acts as a facilitator in Sprint Retrospective and moderator in review and other ceremonies, What are your views? @udeysingh95

File: 7-scrum-master-role/5-moderation.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+**5. Pay attention:** Being an event moderator is hard work! You are the only one who knows for sure that you need to pay attention. You want to know what happens and you want to make sure you ask the right questions. For that, you need to pay attention.\
+**6. Be a host:** Finally, As a moderator, you are more than the person watching time. You are the person in charge of the session. Be attentive. Know who’s in front of you and make sure they have a productive session and that they get something out of the meeting.\
+Now let us look at some characteristics of a great moderator.
+
+## Characteristics of a Great Moderator
+
+Perhaps the decisive achievement of moderation is therefore to move the colleagues in the teams from a technical understanding of the problem to a social understanding of the topics. To create this kind of mindset the SM needs to have these 8 most important qualities of a great moderator.
+
+### 1. Natural curiosity:
```
Comment (Vaibhavpratapsingh22) [inline 7-scrum-master-role/5-moderation.md:37]: Remove the colons ":" from all the headings.

File: 7-scrum-master-role/5-moderation.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+## Moderator
+
+A Moderator is someone who tries to help other people come to an agreement. They help the meeting group to achieve its goals in a focused manner. To guide the discussion in the proper direction, focus groups are typically led by at least one moderator. An event moderator is the master of ceremony of the event. He or she is there to make sure the speakers can do the best job and the audience gets the most out of the day or session. A moderator introduces speakers. He or she also makes sure the speakers stick to the time and the moderator asks and moderate questions. He or she is the connection between the different talks. Having a successful moderator can make a real difference in the effectiveness of a focus group. Let's see Scrum Master, as Moderator.
+
+## Scrum Master as Moderator
```
Comment (Vaibhavpratapsingh22) [inline 7-scrum-master-role/5-moderation.md:9]: Can we add one or two images as there is no image in the whole chapter which looks very boring to me. I would like to see some images. What u say??

**Review by Vaibhavpratapsingh22 — CHANGES_REQUESTED** [review]

File: 7-scrum-master-role/5-moderation.md (hunk 5 above)
Comment (udeysingh95) [reply 7-scrum-master-role/5-moderation.md:7]: sounds fair!

File: 7-scrum-master-role/5-moderation.md (hunk 7 above)
Comment (udeysingh95) [reply 7-scrum-master-role/5-moderation.md:9]: Maybe 1 is ok. Chapter content itself is amusing. Did you get a chance to read the content once @Vaibhavpratapsingh22 ?

Comment (Vaibhavpratapsingh22) [reply 7-scrum-master-role/5-moderation.md:9]: Yes

Comment (SaarthakDeshmukh) [reply 7-scrum-master-role/5-moderation.md:9]: @Vaibhavpratapsingh22 I tried finding an image, but none of them are giving an impact. So can we let this chapter be without an image? as it is a very small chapter. What are your views?

Comment (Vaibhavpratapsingh22) [reply 7-scrum-master-role/5-moderation.md:9]: Yeah, no problem.

**Review by Vaibhavpratapsingh22 — APPROVED** [review]

**Pull request merged**